boilerplate/openshift/golang-osd-operator/csv-generate/csv-generate.sh

```diff
@@ -85,9 +85,21 @@ else
         if [ -z "$DEPLOYED_HASH" ] ; then
             DEPLOYED_HASH=$(
                 curl -s "https://gitlab.cee.redhat.com/service/app-interface/raw/master/data/services/osd-operators/cicd/saas/saas-${operator_name}.yaml" | \
-                    docker run --rm -i quay.io/app-sre/yq:3.4.1 yq r - "resourceTemplates[*].targets(namespace.\$ref==/services/osd-operators/namespaces/hivep01ue1/${operator_name}.yml).ref"
+                    docker run --rm -i quay.io/app-sre/yq:3.4.1 yq r - "resourceTemplates[*].targets(namespace.\$ref==/services/osd-operators/namespaces/hivep01ue1/cluster-scope.yml).ref"
```

Comment:
Did we ascertain that this will work for all the operators? Based on what's in [that directory](https://gitlab.cee.redhat.com/service/app-interface/-/tree/master/data/services/osd-operators/namespaces/hivep01ue1), I'm guessing not -- we would need to condition cluster- vs hive-scoped operators somehow.

Reply:
let me validate.

Reply:
Fixed.